pkg/controller/job/indexed_job_utils.go

```diff
@@ -206,6 +207,15 @@ func addCompletionIndexAnnotation(template *v1.PodTemplateSpec, index int) {
 	template.Annotations[batch.JobCompletionIndexAnnotationAlpha] = strconv.Itoa(index)
 }
 
+func getPodGenerateNameWithIndex(jobName string, index int) string {
```

Comment:
nit: `podGenerateNameWithIndex`

we avoid the word `get` in golang.

Reply:
Done.